convention/src/main/java/com/appcoins/wallet/convention/plugins/AndroidAppPlugin.kt

```diff
@@ -53,7 +53,8 @@ class AndroidAppPlugin : Plugin<Project> {
 
         buildTypes {
           debug {
-            isMinifyEnabled = false
+            isMinifyEnabled = true
+            isShrinkResources = true
```

Comment:
In the debug variant, this is not recommended and may cause some disadvantages such as:

- Slower Build Times, as this two flags add extra processing at build time), 
- Problems with debugging, as obfuscating all the content in debug variant might lead to issues with trying to run in debug and using breakpoints

There are possibly more issues with it, but these two come to mind. 

Also would recommend adding proguard rules to each module, since it helps with app size and its obfuscation